### PR TITLE
Refactored cmd_anal.c to use PJ.c 

### PR DESCRIPTION
**Pull request by deepakchethan**
For axgj command the prev json was `{"34897234":{"refs":[{"2342355":{"type":"fcn...` The address field was displayed as a string. 

Now after refactoring it shows `{4294971308:{"refs":[{4294971215:{"type":"fcn","fcn_addr":4294971200,"name":"entry0","refs":[]}}]}}`  

Should I make it string or keep it as it is?

Edit: I'll just keep it as string

### CODE REVIEW

File: libr/include/r_util/pj.h [hunk 1]
```diff
@@ -26,6 +26,7 @@ R_API PJ *pj_o(PJ *j);
 R_API PJ *pj_a(PJ *j);
 /* keys, values */
 R_API PJ *pj_k(PJ *j, const char *k);
+R_API PJ *pj_Nk(PJ *j, st64 k);
```
Comment (radare) [inline libr/include/r_util/pj.h]: this is not valid json

File: libr/core/cmd_anal.c [hunk 2]
```diff
@@ -5708,9 +5732,14 @@ static void anal_axg (RCore *core, const char *input, int level, Sdb *db, int op
 			if (is_r2) {
 				r_cons_printf ("agn 0x%08"PFMT64x" %s\n", fcn->addr, fcn->name);
 			} else if (is_json) {
-				r_cons_printf ("{\"%"PFMT64u"\":{\"type\":\"fcn\","
-					"\"fcn_addr\":%"PFMT64u",\"name\":\"%s\",\"refs\":[",
-					addr, fcn->addr, fcn->name);
+				pj_o (pj);
+				pj_Nk (pj, addr);
```
Comment (radare) [inline libr/core/cmd_anal.c]: wtf addr is ut64 dont use this api

File: libr/core/cmd_anal.c [hunk 3]
```diff
@@ -5708,9 +5732,14 @@ static void anal_axg (RCore *core, const char *input, int level, Sdb *db, int op
...
+				pj_o (pj);
+				pj_ks (pj, "type", "fcn");
+				pj_kN (pj, "fcn_addr", fcn->addr);
```
Comment (radare) [inline libr/core/cmd_anal.c]: same here

File: libr/core/cmd_anal.c [hunk 4]
```diff
@@ -5722,7 +5751,11 @@ static void anal_axg (RCore *core, const char *input, int level, Sdb *db, int op
 			if (is_r2) {
 				r_cons_printf ("age 0x%08"PFMT64x"\n", fcn->addr);
 			} else if (is_json) {
-				r_cons_printf ("{\"%"PFMT64u"\":{\"refs\":[", addr);
+				pj_o (pj);
+				pj_Nk (pj, addr);
```
Comment (radare) [inline libr/core/cmd_anal.c]: and there

File: libr/core/cmd_anal.c [hunk 5]
```diff
@@ -5740,64 +5773,83 @@ static void anal_axg (RCore *core, const char *input, int level, Sdb *db, int op
 				r_cons_printf ("age 0x%08"PFMT64x" 0x%08"PFMT64x"\n", fcn->addr, addr);
 			} else if (is_json) {
 				if (level == 0) {
-					r_cons_printf ("{\"%"PFMT64u"\":{\"type\":\"fcn\",\"fcn_addr\": %"PFMT64u",\"name\":\"%s\",\"refs\":[", ref->addr, fcn->addr, fcn->name);
+					pj_o (pj);
+					pj_Nk (pj, ref->addr);
+					pj_o (pj);
+					pj_ks (pj, "type", "fcn");
+					pj_kN (pj, "fcn_addr", fcn->addr);
```
Comment (radare) [inline libr/core/cmd_anal.c]: too

**Comment by deepakchethan** [comment]
Done with the changes

File: libr/core/cmd_anal.c [hunk 6]
```diff
@@ -6727,11 +6787,11 @@ static void cmd_agraph_print(RCore *core, const char *input) {
 		r_agraph_foreach_edge (core->graph, agraph_print_edge, NULL);
 		break;
 	case 'J':
-	case 'j':
+	case 'j': {
 		r_cons_printf ("{\"nodes\":[");
```
Comment (xarkes) [inline libr/core/cmd_anal.c]: Why don't you use pj here too?

Comment (deepakchethan) [reply libr/core/cmd_anal.c]: I thought I'd go file wise. But now I made the changes like you said.

**Pull request merged**